readme.md

```diff
@@ -619,6 +687,26 @@ sw-cdu-001 Config Generated
 sw-cdu-002 Config Generated
 sw-leaf-bmc-001 Config Generated
 
+```
+#### Generate Network Config With Overrides
+This option allows you to give pass in a override file and apply it to the desired switches on the network.
+
+The instructions are exactly the same as **[Generate Switch Config with overrides](#generate-switch-config-with-overrides)**
+
```

Comment:
Suggest a sentence here to say that overrides uses hier_config library and it's format.  Might want to link to the hier_config docs for this file.